src/components/rangeFilterInput/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+import {
+  InputGroup,
+  NumberInput,
+  NumberInputField,
+  NumberInputProps,
+} from '@chakra-ui/react';
+
+import Stack from '../stack';
+import InputLeftElement from './InputLeftElement';
+
+type InputElement = 'from' | 'to';
+
+type ChangeCallback = {
+  touched: InputElement;
+  value: {
+    from: number;
+    to: number;
+  };
+};
+
+type RangeFilterInputProps = {
+  name: {
+    from: string;
+    to: string;
+  };
+  handleChange: (event: ChangeCallback) => void;
+  placeholder?: {
+    from: string;
+    to: string;
+  };
+  unit?: string;
+  value: {
+    from: number;
+    to: number;
+  };
+  size?: 'md' | 'lg';
```

Comment:
observation: A lot of props have a 
```
{ 
  from: ...
  to: ...
}
```
Would it make sense to transform the interface to something like
```
{
  from: { 
    name: string;
    value: number;
    placeholder: string;
  };
  to: { 
    name: string;
    value: number;
    placeholder: string;
  }
}
```

Reply:
Yes, Love it! Thanks!